pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc

```diff
@@ -0,0 +1,52 @@
+= Configuring a Network Proxy for PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+If you use a network proxy to connect to the internet, you may have to define the network proxy for the PDK tools. Besides Docker, all tools downloaded in the xref:policies-pdk-prerequisites.adoc[PDK prerequisites] accept the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. 
+
+To configure your environment variable, refer to <<configure-a-network-proxy>>.
+
+To use Docker with a network proxy, also <<configure-a-network-proxy-for-docker>>.
+
+NOTE: Depending on your network proxy, you may have to define the domains the PDK tools access in your proxy's allowlist.
+
+[[configure-a-network-proxy]]
+== Configure a Network Proxy
+
+Configure the environment variables by running following command after replacing the variables with your information:
+
+* `HTTPS_PROXY`:
++
+[source,ssh]
+----
+export HTTPS_PROXY=<proxy-method>://<proxy-server>:<proxy-port>
+----
+* `HTTP_PROXY`:
++
+[source,ssh]
+----
+export HTTPS_PROXY=<proxy-method>://<proxy-server>:<proxy-port>
+----
+
+If your network proxy requires authentication, use the following command:
+
+[source,ssh]
+----
+export HTTPS_PROXY=<proxy-method>://<username>:<password>@<proxy-server>:<proxy-port>
+----
+
+The PDK tools may visit the following domains during execution:
```

Comment:
may > might
Did you think of a better word than "visit"? It is good to respond to a comment if you don't agree with it or if you can't think of a way to resolve it.